packages/create-next-app/index.ts

```diff
@@ -43,6 +43,19 @@ const program = new Commander.Command(packageJson.name)
   .parse(process.argv)
 
 async function run(): Promise<void> {
+  // NodeJs Version Validation
+  var currentNodeVersion: Number = parseFloat(process.versions.node)
+
+  // Throws an Error if NodeJs Version is less than 10.13.x
+  if (currentNodeVersion < 10.13) {
```

Comment:
This value should be imported from package.json's engine entry

Reply:
When I try to import Package.json, it gives an error  `Require stack: ...`.